php-classes/Slate/CBL/Demonstrations/DemonstrationSkillsRequestHandler.php

```diff
@@ -29,8 +29,15 @@ public static function handleBrowseRequest($options = [], $conditions = [], $res
                 return static::throwNotFoundError('Student not found');
             }
 
-            if (!$GLOBALS['Session']->hasAccountLevel('Staff') && $Student->ID != $GLOBALS['Session']->PersonID) {
-                return static::throwUnauthorizedError('Only staff may browse others\' records');
+            if (
+                    !$GLOBALS['Session']->hasAccountLevel('Staff') &&
+                    $Student->ID != $GLOBALS['Session']->PersonID &&
+                    !$GuardianRelationship = \Emergence\People\GuardianRelationship::getByWhere([
```

Comment:
@nbey `use` this class at the top of the file to shorten this line and make clear at the top all the external dependencies